geomesa-plugin/src/main/scala/org/locationtech/geomesa/plugin/wcs/GeoMesaCoverageReader.scala

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2014 Commonwealth Computer Research, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.locationtech.geomesa.plugin.wcs
+
+import com.typesafe.scalalogging.slf4j.Logging
+import org.geotools.coverage.CoverageFactoryFinder
+import org.geotools.coverage.grid.io.AbstractGridCoverage2DReader
+import org.geotools.coverage.grid.{GridCoverage2D, GridEnvelope2D}
+import org.geotools.factory.Hints
+import org.geotools.geometry.GeneralEnvelope
+import org.geotools.referencing.CRS
+import org.geotools.referencing.crs.DefaultGeographicCRS
+import org.geotools.util.Utilities
+import org.joda.time.format.DateTimeFormat
+import org.joda.time.{DateTime, DateTimeZone}
+import org.locationtech.geomesa.raster.data.{AccumuloRasterStore, GeoMesaCoverageQueryParams}
+import org.opengis.parameter.GeneralParameterValue
+
+import scala.util.Try
+
+object GeoMesaCoverageReader {
+  val GeoServerDateFormat = DateTimeFormat.forPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'")
```

Comment:
Another way to specify this format -- ISO8601 -- is using a utility method in Joda:

```scala
val GeoServerDateFormat = ISODateTimeFormat.dateTime()
```

When the timezone is UTC (when the offset is 0), it will yield strings of the form, "2015-06-03T18:22:56.534Z".  Just for fun.